chore: add end-of-life schedule to README (#53)
cdklabs/cdk-assets: README.md

A tool for publishing CDK assets to AWS environments.

<!--BEGIN STABILITY BANNER-->

---

> The AWS SDK for JavaScript v2 will enter maintenance mode on September 8, 2024
> and will reach end-of-support on September 8, 2025. 
>
> The implementation of DefaultAwsClient in cdk-assets is tightly coupled to v2 of the AWS JavaScript SDK and, 

> > as such, will follow the same end-of-life schedule. 

Reviewer: ```suggestion
> therefore, will follow the same end-of-life schedule.
```
